src/pineko/configs.py

```diff
@@ -11,6 +11,18 @@
 configs = {}
 "Holds loaded configurations"
 
+needed_keys = [
```

Comment:
if this is supposed to be a constant make it upper case
```suggestion
NEEDED_KEYS = [
```